common/src/main/resources/assets/quark/lang/es_es.json

```diff
@@ -78,6 +84,8 @@
 	"quark.gui.celebration.wwd": "¡Feliz Día Mundial de la Vida Silvestre!",
 	"quark.gui.celebration.hrd": "¡Feliz Día de los Derechos Humanos!",
 	"quark.gui.celebration.ny": "¡Feliz Año Nuevo!",
+	
+	"quark.gui.celebration.edballs": "Ed bolas",
```

Comment:
"Ed Balls" is a reference to [Ed Balls Day](https://en.wikipedia.org/wiki/Ed_Balls#Ed_Balls_Day). A few of the other lang files translated this, but most left it as `Ed Balls` because it's a person's name.